ArkConfigSettingsCall         req_data
---
ArkConfigSettingsResponse     res_data
bool                          ark_service_timeout
